### ISSUE TITLE
The Binding of Isaac Afterbirth plus go in game but freeze with all possible errors

### ISSUE THREAD

**Issue by greggameplayer** [yuzu-emu/yuzu#204]
## Hardware :
- Nvidia GeForce GTX 960
- AMD FX-6300
- 8 GB Of Ram

## Issues :
The Binding of Isaac Afterbirth Plus now go in game but freeze directly with an infinite list of unmapped Read32 errors and Write64 errors and nvidia not implemented errors too.
I think this will help you a lot.

## a part of the Log :
[Click Here](https://gist.github.com/greggameplayer/b950bd62084ab3a0f55f03c14e48e86f)
I can't put you all the log because the log do about 286939 lines.

**Comment by Hexagon12** [comment]
We don't provide support for booting games up, since there's no public game dumping tools yet.

Requesting to close.

**Comment by greggameplayer** [comment]
Even if it help you to move forward ?

**Comment by Hexagon12** [comment]
Even if it means moving forward. You probably (and most likely) pirated / downloaded the game. We don't support piracy.

**Comment by greggameplayer** [comment]
But for now it's the only way to know more about switch code requirement in terms of games

**Comment by mailwl** [comment]
atm this logs is completely useless, i think. Just wait until yuzu stop crashing before going ingame

**Comment by greggameplayer** [comment]
In some games yuzu don't crash

**Comment by mailwl** [comment]
Like Isaac? It's crash, actually, if it starts from ide

**Comment by greggameplayer** [comment]
and how do you mean by "starts from ide" ?

**Comment by mailwl** [comment]
I mean, if yuzu run from IDE, like Visual Studio, VS stops at assert, and shows more info instead continue executing. Like this:

![2018-02-20](https://user-images.githubusercontent.com/629437/36442020-73caa41a-1685-11e8-8868-19eda09ab07f.png)

**Comment by greggameplayer** [comment]
I don't have test in ide

**Comment by jroweboy** [comment]
> But for now it's the only way to know more about switch code requirement in terms of games

Please understand that the developers are more than aware that just about everything is currently missing. Logs are very misleading as they only show symptoms but not the root of the problem. For instance yuzu can crash in one service, but the actual problem is in the return value from a different service, and the game will use that return value elsewhere to cause the crash. The devs working on the kernel and os are already using commercial games and reverse engineering how they work, and not filling out what the logs says it crashes on. This is much more effective then reading the result of the log as you can find core issues instead of hacking together something that you might think gets the game further.

In short, logs are misleading and not useful at the moment. Unless you are going to take the time to trace the crash and fix it yourself, then there isn't anything that testing games and sending logs can help with for the time being.

**Comment by jroweboy** [comment]
> ok so i will test it under ide and trace the crash if i can

I apologize for not being clear enough. This is not sufficient to figure out whats wrong right now. The entire kernel isn't stable enough to trust crash locations. Its not enough to know that a crash happened, we have to look at why the crash happened and reverse engineer the game code and switch os modules to see where the values came from. That is what I mean by "trace". I know you just want to help, but reporting crashes is not very useful right now. Wait until the emulator is more stable.

**Comment by greggameplayer** [comment]
Ok so i will wait until the core being stable enough

**Comment by greggameplayer** [comment]
Another question, how can i get the shared font of the switch ?

**Comment by LogPod** [comment]
That is copyrighted material and you asking for that is you admitting to piracy. and you dump them from your switch.

**Comment by greggameplayer** [comment]
no, i wanted to say how to dump it from switch because it is written no where

**Comment by mailwl** [comment]
@greggameplayer, dumping shared font from switch is out of scope of this issue. Please, create new one.

**Comment by greggameplayer** [comment]
ok, it's just a simple question not an issue

**Comment by mailwl** [comment]
ah, well, simple answer - just wait till dumping homebrew will appear. similar 3ds

**Comment by greggameplayer** [comment]
shared font is the same as 3ds or completely different ?

**Comment by greggameplayer** [comment]
some errors of this game has been corrected by a PR.

**Comment by mailwl** [comment]
omg! if you even can't get shared font, just... issue tracker for developers, not for random users

**Comment by Hexagon12** [comment]
^. GitHub should only be used to track down issues / regressions. All other talk should be directed to our Discord (since we don't have forums yet). Again, as stated in this thread before, we know of these commercial game issues and what is causing the problems. 

Would be nice if someone cleaned this issue up.
